### DISCUSSION TITLE
hardcopies as vim

### DISCUSSION THREAD

**Discussion by trapalleiro-git** [helix-editor/helix#2521] [Ideas]
command 'ha' to print  hardcopies as vim

**Comment by pickfire** [comment]
Oh, I didn't know there is such thing. But I think you can always save the file and do `lp`.

**Comment by benjaminrich** [comment]
A really useful vim command is `:TOhtml` (it's actually defined in a standard plugin). Then you can print your code with full syntax highlighting in any colorscheme, or put it in a document or email and it looks classy. It would be cool if helix had this as well.
